.ci/macos-steps.yaml

```diff
@@ -25,6 +25,13 @@ steps:
       brew install --cask julia
     fi
 
+    # Install STB.
+    wget https://mlpack.org/files/stb.tar.gz
+        tar -xvzpf stb.tar.gz # Unpack into stb/.
+        cd stb &&\
+        sudo cp -vr include/* /usr/include/ && \
```

Comment:
I wonder if maybe `/usr/include/` isn't the right path to install to on OS X?

Reply:
@zoq Any thoughts? I have no idea how OS X is structured internally

Reply:
The build logs indicate that other libraries are found in `/usr/local/include/`, so maybe you should try there?

```
2021-08-07T17:23:36.2370180Z -- Found cereal: /usr/local/include (found suitable version "1.3.0", minimum required is "1.1.2") 
2021-08-07T17:23:36.7662720Z -- Found Boost: /usr/local/include (found suitable version "1.76.0", minimum required is "1.58")  
```